dist/windows/installer.nsi

```diff
@@ -119,6 +119,18 @@ Section $(inst_startmenu) ;"Create Start Menu Shortcut"
 
 SectionEnd
 
+Section $(inst_startup) ;"Start qBittorrent on Windows start up"
```

Comment:
Sorry for not mentioning this previously. IMO, the setting shouldn't be enabled by default. To achieve that use:
```nsis
Section /o $(inst_startup) ;"Start qBittorrent on Windows start up"
```

Reply:
Added.
Offtopic: Please do not include lib 1.2.4 yet in upcoming qbt. It has got some issues with tracker and incoming connections.